lib/crawl.js

```diff
@@ -100,10 +101,13 @@ function crawl(url, flags) {
             const currentScenario = Object.assign({}, defaultScenario);
             currentScenario.label = queueItem.path;
             if (flags.referenceUrl) {
-                currentScenario.referenceUrl = queueItem.url.replace(
-                    url,
-                    flags.referenceUrl
-                );
+                // Normalize the URL to remove duplicate slashes. See #25
```

Comment:
@danreeves updated to use `normalize-url`. This change is testing well for me on the site where a trailing slash is enforced. Can you test on your site that was creating duplicate slashes?

Reply:
Yep this looks good. Will merge when the tests pass. Not trimming trailing slashes, adding one back on, or updating the fixture to be trailing-slash-less. Up to you.

Reply:
Went with adding a trailing slash back on when the main `url` has one but the `referenceUrl` after `normalize-url` does not. This will be a better fit for sites that enforce a trailing slash, such as the one I've been testing.